airbyte-integrations/base-java/src/main/java/io/airbyte/integrations/base/IntegrationRunner.java

```diff
@@ -71,8 +71,7 @@ public void run(String[] args) throws Exception {
         stdoutConsumer.accept(Jsons.serialize(destination.check(config)));
       }
       case DISCOVER -> {
-        final JsonNode config = parseConfig(parsed.getConfigPath());
-        stdoutConsumer.accept(Jsons.serialize(destination.discover(config)));
+        throw new IllegalStateException("Discover is not implemented for destinations");
```

Comment:
Is this the right thing to do?

Reply:
My goal was to remove friction from adding new integrations. So I think this approach is successful in that regard. If we feel really confident that we don't want to have discover in destinations, i can root it out all the way down to the base. I think we're going to want it though, so I stopped here. Don't feel too strongly though.